docs/source/diagrams/pbft-view-change.puml

```diff
@@ -0,0 +1,129 @@
+@startuml
```

Comment:
It would be great to add a doc with a text description of a View Change process.
It's important to mention in this doc how Primary Selection logic works and how it uses node_reg.